tools/compileProtos.ts

```diff
@@ -104,24 +106,49 @@ async function buildListOfProtos(protoJsonFiles: string[]): Promise<string[]> {
  * @param {string[]} protos List of proto files to compile.
  */
 async function compileProtos(protos: string[]): Promise<void> {
-  const output = path.join('protos', 'protos.json');
+  // generate protos.json file from proto list
+  const JSONoutput = path.join('protos', 'protos.json');
   if (protos.length === 0) {
     // no input file, just emit an empty object
-    await writeFile(output, '{}');
+    await writeFile(JSONoutput, '{}');
     return;
   }
-  const pbjsArgs = [
+  const pbjsArgs4JSON = [
     '--target',
     'json',
     '-p',
     path.join(__dirname, '..', '..', 'protos'),
     '-p',
     'protos',
     '-o',
-    output,
+    JSONoutput,
   ];
-  pbjsArgs.push(...protos);
-  await pbjsMain(pbjsArgs);
+  pbjsArgs4JSON.push(...protos);
```

Comment:
Please add tests that verify added functionality.

Reply:
Sure.